src/components/Display.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ result }) {
+  return (
+    <div>
+      <h2>{result}</h2>
+    </div>
+  )
+};
+
+Display.prototype = {
+  name: PropTypes.String
+};
+
+Display.defaultProps = {
+  result: 0,
```

Comment:
- This should be a string. Change it to `result: '0'`

Reply:
my bad I forget about it thanks for pointing it out, and thanks for the feedback